bin/resources/GameIndex.yaml

```
@@ -20509,6 +20511,11 @@ SLKA-25112:
 SLKA-25115:
   name: "Rockman X7"
   region: "NTSC-K"
+  gameFixes:
+    - SoftwareRendererFMVHack # Flickering and wrong textures in FMV.
+  compat: 5
```

```suggestion
  compat: 5
  gameFixes:
    - SoftwareRendererFMVHack # Flickering and wrong textures in FMV.
```

Compatiblity info should come before the settings themselves.